communication/communications.py

```diff
@@ -146,8 +148,18 @@ def left(self, speed):
   def right(self, speed):
       self.write(TEAM, RIGHT, int(speed))
 
+  def kick(self, speed):
+      self.write(TEAM, KICK, speed)
+
   def grab(self, speed):
       self.write(TEAM, GRAB, speed)
+  
+  def store(self, file):
+      bytes_to_store = len(file)
+      self.write(TEAM, STORE, bytes_to_store)
+      for byte in bytes_to_store:
+          self.to_robot(byte)
```

Comment:
look into using the standardised protocol for sending. We don't want messages going out of sync.
Maybe just using the `STORE` command to send a single byte?